tests/acceptance/test_node_operators_registry.py

```diff
@@ -79,6 +91,10 @@ def test_petrified():
             CURATED_STAKING_MODULE_STUCK_PENALTY_DELAY,
             {"from": contracts.voting},
         )
+    with reverts("CONTRACT_NOT_INITIALIZED"):
```

Comment:
This test verifies NOR contract implementation instead of the proxy, it does not depend on the vote and should remain valid after the vote.  

Please clarify what specifically needs to be cleaned up if I missed something.